### PR TITLE
hle: Some cleanup

### PR DESCRIPTION
**Pull request by AcK77**
This PR cleaned up a bit the HLE folder and the VirtualFileSystem one, since we use LibHac, we can use some class of it directly instead of duplicate things. The "Content" of VFS folder is removed since it should be handled in the NCM service directly.
A larger cleanup should be done later since there is still be duplicated code here and there.

### CODE REVIEW

File: Ryujinx.HLE/FileSystem/ContentPath.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+using Path = System.IO.Path;
+
+namespace Ryujinx.HLE.FileSystem
+{
+    internal static class ContentPath
+    {
+        public const string SystemContent    = "@SystemContent";
+        public const string UserContent      = "@UserContent";
+        public const string SdCardContent    = "@SdCardContent";
+        public const string SdCard           = "@Sdcard";
```
Comment (emmauss) [inline Ryujinx.HLE/FileSystem/ContentPath.cs:16]: I remember there being an issue where `@SdCard` was requested instead. I can't find the issue now as github is currently broken.

Comment (Thog) [reply Ryujinx.HLE/FileSystem/ContentPath.cs:16]: Did you find the issue again?

File: Ryujinx.HLE/HOS/Horizon.cs [hunk 2]
```diff
@@ -238,6 +238,8 @@ public Horizon(Switch device)
             SurfaceFlinger = new SurfaceFlinger(device);
 
             InitializeAudioRenderer();
+
```
Comment (gdkchan) [inline Ryujinx.HLE/HOS/Horizon.cs:241]: I'd remove the separation here.

**Review by gdkchan — APPROVED** [review]
I took a quick look, it looks fine in general.

File: Ryujinx.HLE/FileSystem/ContentPath.cs (hunk 1 above)
Comment (emmauss) [reply Ryujinx.HLE/FileSystem/ContentPath.cs:16]: No. `@SdCard` must have been a mistake on my end, and `@Sdcard` was the actual token.

**Review by Thog — APPROVED** [review]

**Pull request merged**